pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "ConStrain"
+version = "0.3.1"
+description = "ConStrain is a data-driven knowledge-integrated framework that automatically verifies that building system controls function as intended."
+authors = ["Lei, Xuechen <xuechen.lei@pnnl.gov>", "Chen, Yan <yan.chen@pnnl.gov>", "Lerond, Jeremy <jeremy.lerond@pnnl.gov>", "Jung, Yun Joon <yunjoon.jung@pnnl.gov>"]
+readme = "README.md"
+
+[tool.poetry.scripts]
+constrain = "constrain.cli:cli"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+matplotlib = "^3.8.2"
+seaborn = "^0.13.0"
+pandas = "^2.1.3"
+eppy = "^0.5.63"
+fuzzywuzzy = "^0.18.0"
+scikit-learn = "^1.3.2"
+scipy = "^1.11.4"
+tqdm = "^4.66.1"
+uuid = "^1.30"
+pytest = "^7.4.3"
+brickschema = "0.7.1"
```

Comment:
@yunjoonjung-PNNL and I observed that more recent versions trip the Brick validation unit test.